src/frontend/packages/core/src/features/cloud-foundry/users/manage-users/manage-users-confirm/manage-users-confirm.component.ts

```diff
@@ -105,18 +105,34 @@ export class UsersRolesConfirmComponent implements OnInit, AfterContentInit {
     };
   }
 
-  constructor(private store: Store<AppState>, private cfRolesService: CfRolesService, private cfUserService: CfUserService) { }
+  constructor(
+    private store: Store<AppState>,
+    private cfRolesService: CfRolesService,
+    private cfUserService: CfUserService) { }
 
   ngOnInit() {
     this.createCfAndOrgObs();
-
     this.createChangesObs();
+
+    this.isBlocked$ = this.cfGuid$.pipe(
+      filter(res => !!res),
+      map(() => false),
+      startWith(true),
+    );
   }
 
   ngAfterContentInit() {
-    this.cfAndOrgGuid$.pipe(
-      mergeMap(({ cfGuid, orgGuid }) => this.cfRolesService.fetchOrgEntity(cfGuid, orgGuid)),
-    ).subscribe(org => this.orgName$.next(org.entity.name));
+    this.changes$.pipe(
```

Comment:
This could be a little cleaner
```
this.orgName$ = this.changes$.pipe(
      filter((changes) => !!changes.length),
      map((changes) => {
        const orgNames = changes.map((c) => c.orgName);
        return Array.from(new Set(orgNames)).map((orgName) => `'${orgName}'`).join(', ');
      }),
      first(),
    );
```